docker/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN wget http://dl.google.com/android/android-sdk_r24-linux.tgz && \
     mv android-sdk-linux /usr/local/android-sdk
 RUN echo y | /usr/local/android-sdk/tools/android update sdk --filter tools --no-ui -a
 RUN echo y | /usr/local/android-sdk/tools/android update sdk --filter platform-tools --no-ui -a
-RUN echo y | /usr/local/android-sdk/tools/android update sdk --filter build-tools-23.0.0 --no-ui -a
+RUN echo y | /usr/local/android-sdk/tools/android update sdk --filter build-tools-23.0.3 --no-ui -a
```

Comment:
This change here doesn't appear to match anything. That means no build tools are installed and the build fails later on.

Reply:
this line just update build tools to resolve some bugs, what is your meaning build-tools not installed?

I think the problem is CI use https://hub.docker.com/r/rapid7/msf-ubuntu-x64-meterpreter/tags and I do not know how to update that

I think the first step that I update Dockerfile and merge that, after that use them

Reply:
@smcintyre-r7

Reply:
If you means `android update sdk`  does not install build-tools, I trying to test it on Linux and report to you

Reply:
it is very odd, that latest version on linux and windows for build-tools is different, so I change build-tools version in dockerfile to 23.0.1 as you can see in below image

![image](https://github.com/user-attachments/assets/182284df-3529-48b9-92de-ed1373c92833)